uint16 ID
uint32 timestamp
float32 DistX
float32 DistY
uint8 index
float32 VrelX
float32 VrelY
float32 RCS
float32 Lifetime
